Update credits.md (#6107)
circleci/circleci-docs: jekyll/_cci2/credits.md

## Free plan
{: #free-plan }

> As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free Plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.

Reviewer: ```suggestion
As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.
```
I believe we're capitalizing as follows, for consistency: "Free plan," "Performance plan," "Scale plan"

> As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free Plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast. 

Reviewer 2: @thomasr8 lgtm!

The free usage-based plan offers a set amount of build credits across medium-type machines per week (which offers 2 CPUs, 4gb of memory). With this combination, a small number of credits are charged per minute and there is no limit on user seats. Refer to the [Pricing](https://circleci.com/pricing/) page for more information on credit amounts.

> The table below lists some of the features you will be able to use on the Free Plan.

Reviewer: Same as the suggestion regarding keeping "plan" in lowercase

| Feature | Description |
|-------|-------------|

> | UNLIMITED USERS | You may have an unlimited number of users. This enables you to have different teams and groups working together on projects |

Reviewer: I'm wondering if the descriptions should just be full sentences with periods?

> | EXPANSIVE LIST OF AVAILABLE RESOURCE CLASSES | Docker, Linux and Windows are available (macOS coming soon) are available for us |

Reviewer: ```suggestion
Docker, Linux and Windows (macOS coming soon) are available for us |
```
"are available" was repeated twice. Also, is it supposed to be "for use" (and not "for us")?

A bit nitpicky, but I prefer not to have the same word repeated too closely together, so I may remove "available" from the Feature column here i.e. "EXPANSIVE LIST OF RESOURCE CLASSES" - or, rephrase the description.

| CONCURRENCY AND TEST SPLITTING | Faster builds and test splitting equals increased productivity and speed when building on CircleCI |
| SELF HOSTED RUNNERS | Increased number of runners you can use when working on CircleCI |

> | FLAKY TEST DETECTION | Improved inisghts are available to help analyze and detect flaky tests |

Reviewer: ```suggestion
| FLAKY TEST DETECTION | Improved insights are available to help analyze and detect flaky tests |
```

> | UNLIMITED PRIVATE ORBS | Streamline team collaboration by sharing CircleCI Orbs across teams and groups |

Reviewer: ```suggestion
| UNLIMITED PRIVATE ORBS | Streamline team collaboration by sharing CircleCI orbs across teams and groups |
```
I think we don't capitalize features either

> | DOCKER LAYER CACHING | Efficiency features designed specifically for Docker, providing improved speed and efficiency |

Reviewer: ```suggestion
| DOCKER LAYER CACHING | Features designed specifically for Docker, providing improved speed and efficiency |
```
Could we remove the first "efficiency?"

{: class="table table-striped"}

### Unlimited users
{: #unlimited-users }

> When you use the free plan, you may have an unlimited number of users building and developing on CircleCI. This allows you to collaborate with more teams and groups within your organization, which can increase speed, productivity, and efficiency.

Reviewer: ```suggestion
When you use the Free plan, you may have an unlimited number of users building and developing on CircleCI. This allows you to collaborate with more teams and groups within your organization, which can increase speed, productivity, and efficiency.
```
On first read it wasn't super clear to me whether this was saying "Yes, you can have an unlimited number of users" or "You _might_ have an unlimited number of users"...

### Expansive list of available resource classes
{: #expansive-list-of-available-resource-classes }

> When using the free plan, you have the widest array of resource classes on Docker, Linux, Windows and macOS (coming soon) available to use. With this flexibility in choosing resource classes, you can be sure to choose the right compute resources for your job.

Reviewer: ```suggestion
When using the Free plan, you have the widest array of resource classes on Docker, Linux, Windows and macOS (coming soon) available to use. With this flexibility in choosing resource classes, you can be sure to choose the right compute resources for your job.
```
I think I'd also shorten the second sentence to just something like, "This flexibility helps ensure that you are choosing the right compute resources..." 

Also are we referring to `job` as in the CircleCI concept of job?

For more information about these resources, please refer to the the [Executors and Images](https://circleci.com/docs/2.0/executor-intro/) page.

### Concurrency and test splitting
{: #concurrency-and-test-splitting }

> The ability to run multiple tests at the same time (concurrently) is a very powerful feature that allows you to decrease your build times and shorten feedback cycles by running jobs concurrently with workflows. Because one of the most important contributors to developer efficiency is being able to retrieve test results quickly, CircleCI can intelligently split tests to get you your test results faster using test splitting.

Reviewer: ```suggestion
The ability to run multiple tests at the same time (concurrently) allows you to decrease your build times and shorten feedback cycles by running jobs concurrently with workflows. 
```
I think I'd also just remove the second sentence entirely... To me it seems to convey the same message as the preceding sentence.

For more information about how to utilize test splitting, refer to the [Running Tests In Parallel](https://circleci.com/docs/2.0/parallelism-faster-jobs/#using-test-splitting-with-python-django-tests) page.

### Self hosted runners
{: #self-hosted-runners }

CircleCI runners allow you to use your own infrastructure for running jobs, providing more granular control of your own environment and flexibility in building and testing on a wide variety of architectures.

For more information about using CircleCI runners, please see the [CircleCI Runner Overview](https://circleci.com/docs/2.0/runner-overview/) page.

### Flaky test detection
{: #flaky-test-detection }

> Flaky test detection is an important tool that helps you determine why a test may pass one time, and then fail the next time. Because the average failed workflow wastes approximately 30 minutes, having the ability to identify flaky tests can save you significant amounts of time in your builds and workflows.

Reviewer: ```suggestion
The average failed workflow wastes approximately 30 minutes. Identifying flaky tests can save you significant amounts of time in your builds and workflows.
```

For more detailed information about flaky tests, refer to the [Test Insights](https://circleci.com/docs/2.0/insights-tests/#flaky-tests) page.

### Unlimited private orbs
{: #unlimited-private-orbs }

CircleCI orbs are shareable configuration packages that enable developers to create private, standardardized configurations that can be used across an organization by different teams.

For more information on private orbs, refer to the [Orbs Introduction](https://circleci.com/docs/2.0/orb-intro/#private-orbs-vs-public-orbs) page.

### Docker layer caching
{: #docker-layer-caching }

> Reducing the time it takes to build a Docker images is an important consideration for developers who consistently use these images in their workflows.  Docker Layer Caching (DLC) is a great feature to use if building Docker images is a regular part of your CI/CD process. DLC will save image layers created within your jobs, rather than impact the actual container used to run your job.

Reviewer: ```suggestion
Reducing the time it takes to build a Docker image is an important consideration for developers who consistently use these images in their workflows. Consider using Docker Layer Caching (DLC) if building Docker images is a regular part of your CI/CD process. DLC saves image layers created within your jobs, rather than impact the actual container used to run your job.
```
